MapleServer2/Commands/Game/HomeCommands/BallCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Maple2Storage.Types;
+using MapleServer2.Commands.Core;
+using MapleServer2.Constants;
+using MapleServer2.Managers;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using MapleServer2.Types;
+
+namespace MapleServer2.Commands.Game.HomeCommands;
+
+public class BallCommand : InGameCommand
+{
+    public BallCommand()
+    {
+        Aliases = new()
+        {
+            "hostball"
+        };
+        Usage = "hostball [value]";
+        Description = "Changes the gravity of the map.";
```

Comment:
Hmm are these strings necessary? I thought commands like these are already built in the client, just like doing "/?" etc.
Also, shouldn't these descriptions be in the xml strings? We should use that instead rather than hardcoding it.

Reply:
I thought these commands were hidden? Can you see in the client somewhere?